packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -797,6 +1033,45 @@ void registerPlugins(Registrar registrar) {
 }
 ''';
 
+// TODO(egarciad): Evaluate merging the web and desktop plugin registry templates.
```

Comment:
This needs a language version header. Because it isn't under `lib/` it isn't a part of any package and will take whatever the current default language version is. That may be incorrect for users that have not yet migrated to null safety.

In this case, you should pick whatever the language version of the main entrypoint is.

Reply:
good point

Reply:
Done. It now sets the language version returned from `determineLanguageVersion`.